### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by MaggieSchuetz**
Improved styling - please be very honest, right now I can still change it!

### CODE REVIEW

File: src/components/shared/NavBar.js [hunk 1]
```diff
@@ -21,32 +21,30 @@ const StyledNavBar = styled.nav`
...
+      <StyledNavLink
+        to="/newentry"
+        className={isActive => 'nav-link' + (isActive ? ' active' : '')}
+        aria-label="newEntry"
+      >
+        <IoAddCircleOutline alt="New Entry" />
+      </StyledNavLink>
+      <StyledNavLink
+        to="/journalentries"
+        className={isActive => 'nav-link' + (isActive ? ' active' : '')}
```
Comment (mbosselmann) [inline src/components/shared/NavBar.js:42]: As far as I see, you don't have a class `nav-link` in your styled component.
And `NavLink` sets `active` as class automatically. You don't need to implement this mechanism. [See React Router Docs.](https://reactrouter.com/docs/en/v6/api#navlink)

**Review by mbosselmann — COMMENTED** [review]
Looks good! 👍 But I have some remarks you should think about. :)

File: src/components/Header.js [hunk 2]
```diff
@@ -5,18 +5,22 @@ const StyledHeader = styled.div`
   padding: 10px;
   margin: 0;
   text-align: center;
-  color: palevioletred;
   position: sticky;
   top: 0;
-  height: 60px;
-  //background: url('https://images.unsplash.com/photo-1505118380757-91f5f5632de0?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=926&q=80') no-repeat;
+  height: 70px;
+  /* background: url('https://images.unsplash.com/photo-1452421822248-d4c2b47f0c81?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1074&q=80')
```
[the remark this answers is not in the document]
Comment (MaggieSchuetz) [reply src/components/Header.js:11]: I am still figuring out if I want to keep the styling as is or use this background image, so I wanted to leave it in for now.

File: src/components/Form.js [hunk 3]
```diff
@@ -151,34 +157,62 @@ const FormContainer = styled.form`
   flex-direction: column;
   justify-content: center;
   align-items: center;
-  background-color: #fff;
+  background-color: snow;
   color: #333;
-  border: 1px solid black;
+  /* border: 1px solid black; */
```
[the remark this answers is not in the document]
Comment (MaggieSchuetz) [reply src/components/Form.js:162]: Removed

File: src/App.js [hunk 4]
```diff
@@ -92,5 +92,5 @@ function App() {
 export default App;
 
 const AppContainer = styled.div`
-  height: 100vh;
+  /* height: 100vh; */
```
[the remark this answers is not in the document]
Comment (MaggieSchuetz) [reply src/App.js:95]: Removed

File: src/components/EntryList.js [hunk 5]
```diff
@@ -21,7 +21,7 @@ function EntryList({ content, handleDelete, editJournalEntry }) {
 }
 
 const ListContainer = styled.div`
-  height: 100vh;
+  /* height: 100vh; */
```
[the remark this answers is not in the document]
Comment (MaggieSchuetz) [reply src/components/EntryList.js:24]: Actually I did need this, so I uncommented it :)

File: src/components/Form.js [hunk 6]
```diff
@@ -151,34 +157,62 @@ const FormContainer = styled.form`
...
   font-size: 12pt;
   color: grey;
-  padding: 5px;
+  padding: 20px 5px 5px 5px;
+  background-color: white;
+  border: 0;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.2), 0 3px 10px 0 rgba(0, 0, 0, 0.19);
+
+  &:focus {
+    outline: none;
```
[the remark this answers is not in the document]
Comment (MaggieSchuetz) [reply src/components/Form.js:206]: Added an outline and a differently-colored shadow, so now focus should be visible.

File: src/components/Form.js [hunk 7]
```diff
@@ -151,34 +157,63 @@ const FormContainer = styled.form`
...
 
 const Textarea = styled.textarea`
   width: 70vw;
-  height: 40vh;
+  height: 50vh;
   font-family: Open-Sans, Helvetica, Sans-Serif;
   font-size: 12pt;
   color: grey;
-  padding: 5px;
+  padding: 20px 5px 5px 5px;
```
Comment (MichiBerger) [inline src/components/Form.js:200]: typo again ;) but it´s not necessary!

**Review by JelenadlS — APPROVED** [review]
Looks all logical 😊

File: src/components/shared/NavBar.js [hunk 8]
```diff
@@ -27,11 +28,17 @@ const StyledNavLink = styled(NavLink)`
     color: snow;
   }
 `;
-function NavBar() {
+function NavBar({ setEntryEdit }) {
+  const handleClick = e => {
+    setEntryEdit({
+      edit: false,
+    });
+  };
```
Comment (PReichetanz) [inline src/components/shared/NavBar.js:36]: You are mutating your state here which we do not want to do. Additionally, your `item` key gets lost if you do it this way ;)

**Pull request merged**